pkg/mcs/scheduling/server/server.go

```diff
@@ -540,14 +540,6 @@ func (s *Server) GetConfig() *config.Config {
 	cfg.Schedule = *s.persistConfig.GetScheduleConfig().Clone()
 	cfg.Replication = *s.persistConfig.GetReplicationConfig().Clone()
 	cfg.ClusterVersion = *s.persistConfig.GetClusterVersion()
-	if s.storage == nil {
-		return cfg
-	}
-	sches, configs, err := s.storage.LoadAllSchedulerConfigs()
-	if err != nil {
-		return cfg
-	}
-	cfg.Schedule.SchedulersPayload = schedulers.ToPayload(sches, configs)
```

Comment:
Do we need to remove?https://github.com/tikv/pd/blob/f9c2366411257f17edbd822b8c9689621bbfa511/tests/integrations/mcs/scheduling/api_test.go#L428